objects/obj_Message7/Draw_64.gml

```diff
@@ -3,9 +3,10 @@
 var xx = obj_Global.GUI_WIDTH * .5;
 var yy = obj_Global.GUI_HEIGHT - 100;
 
-var text = currentMessage;
-var index = floor(clamp(characterCounter, 0, currentMessageLength));
-var printText = string_copy(text, 1, index);
+//you can manually send in the modified text, but this doesn't play nice with emphasized text.
+//var text = currentMessage;
+//var index = floor(clamp(characterCounter, 0, currentMessageLength));
+//var printText = string_copy(text, 1, index);
```

Comment:
Can you help me understand what these changes were for?  I don't see any use of `characterCounter` in this file on `72` below.

Reply:
When using the `characterCounter` provided, you'll have brief blips where there are incomplete emphases. For instance:
`*Chipot` or, worse, `*Chipotle is some of the best food in the world. Did you know the more *`.
By passing `len`, and determining what the plaintext is, we can remove any and all cases of junk input by:
1. beginning to draw the full string
2. once enough characters are drawn, stop drawing.
I'll remove the unused code.